### PR TITLE
Add events.py with getting events from Google calendar via APIs; add …

### PR DESCRIPTION
**Pull request by DanielaNy**
…events route to Flask

### CODE REVIEW

File: backends/.gitignore [hunk 1]
```diff
@@ -127,3 +127,11 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# credentials
+*client_secret.json
+*credentials.pickle
+*credentials_with_refresh_token.pickle
```
Comment (filias) [inline backends/.gitignore]: you can use this `*.pickle`

File: backends/events.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function
```
Comment (filias) [inline backends/events.py:1]: this is probably for compatibility with python2.7, I think you can use the regular print function.

File: backends/events.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    if not events:
+        print('No upcoming events found.')
+    for event in events:
+        start = event['start'].get('dateTime', event['start'].get('date'))
+        print(start, event['summary'])
+
+    return events
+
+
+if __name__ == '__main__':
```
Comment (filias) [inline backends/events.py:73]: let's remove this.

File: backends/flask_main.py [hunk 4]
```diff
@@ -14,3 +15,9 @@ def get_menu():
 def get_about():
     pyladies_about_info = get_pyladies_about_info()
     return {"pyladies_about_info": "From international Pyladies website:"+ pyladies_about_info}
+
+
+@app.route("/events")
+def get_events():
+    events = get_google_cal_events()
+    return {"pyladies_events": events}
```
Comment (filias) [inline backends/flask_main.py]: Use just `events` as the key of the json.

**Comment by filias** [comment]
I left a few comments, nothing major. Fix that, we merge and then we can improve it more later :)

File: backends/events.py (hunk 2 above)
Comment (filias) [reply backends/events.py:1]: In fact let's just remove the import and the calls to `print` we won't need them.

**Comment by DanielaNy** [comment]
Changed according to comments. Can we merge?

**Review by filias — APPROVED** [review]

**Pull request merged**